examples/transactions_example/visualizer.go

```diff
@@ -0,0 +1,456 @@
+/**
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+// This is the visualiser of intersection states, drawing
+// frame with each intersection observed in the output topic, displaying
+// the current number of cars and traffic light color per road.
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/gdamore/tcell"
+	"os"
+	"sort"
+	"sync"
+	"time"
+)
+
+// height and width per intersection frame
+const hpi = 20
+const wpi = 39
+
+var screen tcell.Screen
+
+type drawPos struct {
```

Comment:
Add documentation clarifying if this is left-right or right-left and bottom-top or top-bottom.